app-games/src/main/java/com/aptoide/android/aptoidegames/categories/presentation/CategoryDetailView.kt

```diff
@@ -31,10 +32,9 @@ import com.aptoide.android.aptoidegames.toolbar.AppGamesTopBar
 import cm.aptoide.pt.aptoide_ui.animations.animatedComposable
```

Comment:
rebase and change to the animatedComposable function on the extensions module

Reply:
This change was in an old commit from an already merged PR